Automattic/newspack-plugin
# Setup Wizard fixes

### All Submissions:






### Changes proposed in this Pull Request:



- Fixes a bug which prevented "Country" field from being displayed in the Setup wizard (Closes https://github.com/Automattic/newspack-plugin/issues/1299)
- Fixes a font picker issue (Closes https://github.com/Automattic/newspack-plugin/issues/1303)
- Closes #1298
- Fixes a bug with Mailchimp integration helper text
- Fixes a bug which caused Homepage overwrite when editing design settings
- Closes https://github.com/Automattic/newspack-plugin/issues/1302

### How to test the changes in this Pull Request:

1. Setup wizard
    1. On `master`, observe the "Country" field missing on Settings step
    2. Switch to this branch, observe the "Country" field is there 
    3. Observe https://github.com/Automattic/newspack-plugin/issues/1302 is resolved
    3. Again on `master`, observe that on a fresh site (or with theme mods reset), the font picker in the "Typography" section of the Design step will start off with "Custom" font choices 
    4. Switch to this branch, observe the font picker starts off with "Default" font choices
    1. Observe all Integrations step items are optional and setup can be proceeded with without configuring Jetpack & Site Kit
    2. Get to that setup wizard once again (visit `/wp-admin/admin.php?page=newspack-setup-wizard#/integrations`)
    3. Observe that Mailchimp cannot be connected until Jetpack is configured and displays a notice to that effect
    3. Connect Jetpack and observe Mailchimp can now be "connected" – via wordpress.com settings
2. Design wizard
    1. On `master`, observe that saving settings in Design wizard overwrites the homepage (creates a new revision)
    2. Observe it's not the case on this branch

### Other information:
